wagtailio/templates/base.html

```diff
@@ -168,15 +149,7 @@
 
         <script src="https://platform.twitter.com/widgets.js" async></script>
         <script async defer src="https://buttons.github.io/buttons.js"></script>
-
-        {% compress js %}
-            <script src="{% static 'js/vendor/jquery.min.js' %}"></script>
-            <script src="{% static 'js/vendor/js-cookie.min.js' %}"></script>
-            <script src="{% static 'js/vendor/owl.carousel.js' %}"></script>
-            <script src="{% static 'js/vendor/jQuery.headroom.min.js' %}"></script>
-            <script src="{% static 'js/vendor/headroom.min.js' %}"></script>
-            <script src="{% static 'js/main.js' %}"></script>
-        {% endcompress %}
+        <script src="{% manifest 'js/main.js' %}"></script>
```

Comment:
Note: I would prefer to add this on a block of ` {% block main_js %}{% endblock main_js %}`` as well as keeping the `extra_js` block so we can invalidate the base JS easier if needed but it could be done later.

Reply:
I'm happy either way. I'm wondering how often we'll need to invalidate the base JS?

Reply:
It can be done later if needed